argocd/overlays/moc-infra/projects/okd-wg.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: okd-wg
+  labels:
+    project-template: global
+spec:
+  destinations:
+    - namespace: 'okd-*'
```

Comment:
For now, okd-wg members should only get access to the okd-wg ns, while okd-team should get access to both okd-wg and okd-wg.
Project-admin of centos and kvm-device-plugin ns should also be with okd-team for now, not okd-wg.

Reply:
so essentially s/okd-wg/okd-team/g here

Reply:
updated